src/sentry/static/sentry/app/views/settings/components/dataScrubbing/content.tsx

```diff
@@ -6,23 +6,23 @@ import EmptyMessage from 'app/views/settings/components/emptyMessage';
 import {IconWarning} from 'app/icons';
 
 import RulesList from './rulesList';
-import DataPrivacyRulesModal from './dataPrivacyRulesModal';
+import Dialog from './dialog';
```

Comment:
Can we keep this as `Modal`? We don't really use `Dialog` anywhere else

Reply:
the Problem is that I am already using the import import Modal from 'react-bootstrap/lib/Modal'; and according to our rules, I should not use default as ...  I will later on use the GlobalModal and change it 😉